nrf_security/doc/backend_config.rst

```diff
@@ -190,49 +172,27 @@ Feature support
 |             |                   | 256-bit key |                       |
 +-------------+-------------------+-------------+-----------------------+
 
+.. note:
+   When the Arm CryptoCell cc3xx backend is enabled, AES GCM is provided by original Mbed TLS implementation.
```

Comment:
Doesn't this note belong in the section where we document the AES GCM support?

Reply:
Fixed